orchagent/routeorch.cpp

```diff
@@ -466,6 +466,28 @@ void RouteOrch::doTask(Consumer& consumer)
             vector<string> ipv = tokenize(ips, ',');
             vector<string> alsv = tokenize(aliases, ',');
 
+            /*
+             * To check the validity of nexthop and make sure it works even in
+             * abnormal cases. One case is old format - empty "nexthop",
+             * then set ip to zero for backward compatibility.
+             */
+            if (alsv.size() == 0)
+            {
+                it = consumer.m_toSync.erase(it);
+                continue;
+            }
+            else if (alsv.size() != ipv.size())
+            {
+                ipv.resize(alsv.size());
+            }
+            for (auto &ip : ipv)
```

Comment:
please have a empty line above. the for is not part of if else block. have a separate line make it clear.